src/vanilla/Model/MethodTS.cs

```diff
@@ -738,6 +763,49 @@ private static void GenerateRequestParameterPath(TSObject parent, string propert
             }
         }
 
+        public string GenerateResponseType(string emptyLine)
+        {
+            TSBuilder builder = new TSBuilder();
+            builder.ExportInterface(HttpOperationResponseName, "msRest.BaseHttpOperationResponse", iface =>
```

Comment:
Use the constant defined above instead of the string literal for "msRest.BaseHttpOperationResponse".